### PR TITLE
Add Windows Instructions to BuildingAndRunning.md

### PR DESCRIPTION
**Pull request by TatianaKapos**
While working in the hermes-windows repository, there was a lack of current documentation needed to run hermes. This PR hopes to mitigate this issue for the next person onboarding into this repository.

Let me know if there's a better place for this! :)

Added to bottom of page to make integrations easier

image

### CODE REVIEW

File: doc/BuildingAndRunning.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ On Arch Linux:
 On Mac via Homebrew:
 
     brew install cmake git ninja
+    
+On Windows:
+
+    You will need to install Visual Studios with following Spectre-migrated libraries
+    You can find these in the Visual Studio Installer under Individual components
+    > MSVC v142 - VS 2019 C++ x64/x86 Spectre-migrated libs (Latest)
```
Comment (jonthysell) [inline doc/BuildingAndRunning.md:30]: I think if you want to build ARM64 you also need:
```suggestion
    > MSVC v142 - VS 2019 C++ ARM64 Spectre-migrated libs (Latest)
    > MSVC v142 - VS 2019 C++ x64/x86 Spectre-migrated libs (Latest)
```

File: doc/BuildingAndRunning.md [hunk 2]
```diff
@@ -54,6 +62,11 @@ The above instructions create an unoptimized debug build. The `--distribute` fla
 
 ## Building on Windows
 
+You should be able to build hermes by running the following script from the base of the repository.
```
Comment (jonthysell) [inline doc/BuildingAndRunning.md:65]: Maybe it's better to just inject a separate header like `## Building on Windows (hermes-windows)` to clarify the instructions for building our fork.

File: doc/BuildingAndRunning.md [hunk 3]
```diff
@@ -54,6 +62,11 @@ The above instructions create an unoptimized debug build. The `--distribute` fla
...
+Note: There may be errors shown on the script but hermes is built if you can find the executable in build/bin/hermes/Debug/hermes.exe
+    
+    .ado\scripts\cibuild.ps1
+
 The Windows build depends on which particular combination of GitBash/Cygwin/WSL and Visual Studio is used.
 
     git -c core.autocrlf=false clone https://github.com/facebook/hermes.git
```
[the remark this answers is not in the document]
Comment (jonthysell) [reply doc/BuildingAndRunning.md:72]: Maybe it makes sense to minimize how much we modify their docs in our fork, to make integrations easier?

**Review by jonthysell — APPROVED** [review]
The content is good, I'm just not sure about where to put it. I don't think we necessarily need to inject stuff in all of the upstream docs - maybe we consolidate our changes under "this is our fork for RNW" section of the readme.

File: doc/BuildingAndRunning.md (hunk 2 above)
Comment (TatianaKapos) [reply doc/BuildingAndRunning.md]: I forgot about integrations, thanks for the reminder! Reformatted just to add our documentation at the bottom of the file :)

**Review by jonthysell — APPROVED** [review]

File: doc/BuildingAndRunning.md [hunk 4]
```diff
@@ -103,3 +103,21 @@ In addition to `hermes`, the following tools will be built:
 - `hbcdump`: Hermes bytecode disassembler
 - `hermesc`: Standalone Hermes compiler. This can compile JavaScript to Hermes bytecode, but does not support executing it.
 - `hvm`: Standalone Hermes VM. This can execute Hermes bytecode, but does not support compiling it.
+
+## Building on Windows (hermes-windows repository)
+
+### Dependencies
+
+    You will need to install Visual Studios with following Spectre-migrated libraries
```
Comment (vmoroz) [inline doc/BuildingAndRunning.md:111]: "Studio" - no plural 's' in the end.

**Review by vmoroz — APPROVED** [review]
LGTM

**Pull request merged**